README.md

```diff
@@ -58,12 +57,40 @@ docker-compose build web
 docker-compose run --rm web rake argo:repo:load
 ```
 
-## Load and index records
+### Load and index records
 
 ```
 docker-compose run --rm web rake argo:repo:load
 ```
 
+## Run the servers (for development)
+
+Spin up the services Argo depends upon, but *not* Argo itself; you will run Argo outside of containerland for maximum agility.
```

Comment:
You certainly don't have to do it this way.  This seems *way* more complicated to me.  Can this be a different commit?

Reply:
I'm glad for this info, but agree a separate PR would be better.

Reply:
@jcoyne If there's an easier way to spin up a fully functional Argo that immediately reflects changes in `app/`, I am all ears. When running the web app in containerland, I recall having to restart and/or rebuild to see changes which is less than ideal.